document/chapters/preliminaries.tex

```diff
@@ -1,3 +1,190 @@
 % !TEX root = ../main.tex
 \chapter{Preliminaries}
 \label{chapter:preliminaries}
+
+Rust is a systems programming language focused on speed, memory safety and concurrency. It intends to offer both performance similar to C++ and memory safety similar to Haskell. Rust is a compiled language having imperative and functional programming features. Its performance makes Rust an ideal candidate for writing operative systems, databases, compilers and other high-performance software without worrying about manual memory allocation. However, its safety and high-level abstractions has encouraged its usage in backend and even frontend development. 
+
+Rust 1.0 was released on May, 2015 after 5 years of initial development lead by Graydon Hoare. During 2013, Hoare stepped down as technical lead on Rust and the language became endorsed by Mozilla. Currently, several companies have written in Rust part of their core applications as can be seen in \href{https://www.rust-lang.org/en-US/friends.html}{Rust Friends website}.
```

Comment:
Rust 1.0 was released on May, 2015 after 5 years of initial development lead by Graydon Hoare. During 2013, Hoare stepped down as technical lead on Rust and the language became endorsed by Mozilla. 
``` There is a disconnect between these two ideas. I guess this is to show the maturity of the language, but the jump from a guy doesn't work there anymore to people use it is too big. You need a glue text sentence in between these two to actually talk about the maturity of the language to conclude that is at a stage where people use it.```
Currently, several companies have written in Rust part of their core applications as can be seen in ```suggestion\href{\url{https://www.rust-lang.org/en-US/friends.html}}```{Rust Friends website}.

Reply:
I divided that paragraph in two: One about Rust's history (In fact one of the core team members sent me an ACM talk about it and I have cited it), and the other one about companies using Rust. I added an small part about the core team and semantic versioning (as the glue text).

What's the difference between `\href{\url{www.example.com}}{Example}` and `\href{www.example.com}{Example}`?